md/bonita-as-windows-service.md

```diff
@@ -0,0 +1,100 @@
+# How to install Bonita as a service on Windows
+
+If you are a Windows server administrator you know how useful can be to have softwares installed as a windows service.
+In this tutorial we show how to install Tomcat with Bonita configured as a windows service.
+
+## Software needed
+
+For this tutorial you will need the following softwares:
+* Tomcat
+* Bonita deploy Bundle
+* Postgresql Database (or another database supported by Bonita)
+Please verify the version supported from the Support page
```

Comment:
⚠️ missing break line
⚠️ wording:  should be `supported version` instead of `version supported `

Reply:
I have changed "version supported" in "supported version" and added a break line